CONTRIBUTING.md

```diff
@@ -53,11 +54,8 @@ There are some external applications loading resources from http://sourced.tech/
 - `https://sourced.tech/img/logos/favicon.png`
```

Comment:
Since the goal of this section is to mention which Landing resources are being hotlinked by Pardot pages, I think we should also add:
```
https://sourced.tech/fonts/cerebrisans-medium.woff
```
which is currently affected by https://github.com/src-d/landing/issues/450